### ISSUE TITLE
Focus indicator is not visible

### ISSUE THREAD

**Issue by zhr1130** [jakezatecky/react-checkbox-tree#281]
Focus indicator is not visible on checkbox tree and due to this, user is unable to see the keyboard focus on which node.
Please see below gif for repro.
![checkbox-tree-indicator](https://user-images.githubusercontent.com/18317674/119249264-04f4e900-bb4c-11eb-83f5-c4efd4ab0204.gif)

**Comment by jakezatecky** [comment]
Thanks for the report.
